activities.go

```diff
@@ -3,32 +3,14 @@ package subscribe_emails
 
 import (
 	"context"
+	"fmt"
 
 	"go.temporal.io/sdk/activity"
 )
 
-type Activities struct {
-}
-
 // email activities
-func (a *Activities) SendWelcomeEmail(ctx context.Context, emailInfo EmailInfo) (string, error) {
-	activity.GetLogger(ctx).Info("sending welcome email to customer", "EmailAddress", emailInfo.EmailAddress)
-	return "Sending welcome email completed for " + emailInfo.EmailAddress, nil
+func SendEmail(ctx context.Context, emailInfo EmailDetails) (string, error) {
+	activity.GetLogger(ctx).Info("Sending email to the following customer: %v", emailInfo.EmailAddress)
```

Comment:
The previous way the log was written is better. This log statement doesn't support `%` formatting.